lib/ruby-debug-ide/attach/native_debugger.rb

```diff
@@ -38,8 +40,8 @@ def find_attach_lib(debase_path)
     raise 'Could not find attach library'
   end
 
-  def attach_to_process(pid)
-    execute "attach #{pid}"
+  def attach_to_process
+    execute "attach #{@pid}"
```

Comment:
For gdb your approach doesn't work without '-p' before the pid, so we have to use a different implementation for lldb and gbd. Therefore, it seems to me that it is better to leave it as it is